src/js/view/view.js

```diff
@@ -278,15 +278,22 @@ define([
                 '.jw-thumb:after'
             ];
 
-            var iconHovers = ['.jw-icon:hover'];
+            var iconHovers = ['.jw-icon:hover', '.jw-option:hover'];
             var sliders = ['.jw-progress'];
             var containers = ['.jw-container'];
 
+
             addStyle('color', iconHovers, _model.get('skinColorActive'));
             addStyle('color', icons, _model.get('skinColorInactive'));
             addStyle('background', cues, _model.get('skinColorInactive'));
             addStyle('background', sliders, _model.get('skinColorActive'));
             addStyle('background', containers, _model.get('skinColorBackground'));
+
+            addStyle('color', ['.jw-active-option'], _model.get('skinColorActive'));
+            addStyle('background', ['.jw-active-option'], _model.get('skinColorInactive'));
+
+            addStyle('color', ['.jw-icon-hd', '.jw-icon-cc'], _model.get('skinColorActive'));
```

Comment:
There is something loopy going on here, 

on line 287 we say "color icons the inactive color"
on line 295 we say "these two icons should be colored the active color"
on line 296 we say "color icons the inactive color if they have the class off"

One solution is to use an "on" class instead of an "off" class. What do you think?